### PR TITLE
Reworks Ravager's eviscerating charge and ravage abilities

### PR DESCRIPTION
**Pull request by ma44**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: ma44
balance: Eviscerating charge now plows through humans, slicing and knocking them around. Ravagers suffer slowdown upon hitting humans no more, nor to they bounce upon hitting dense turfs.
tweak: Ravage no longer has a two tile range limitation for activating the ability, it will also no longer move you one tile in a direction. Using the ability will now make you face the general direction of the mouse and slice all nearly marines in that direction. 
tweak: Ravage can now also hit more than three humans; the rage gained from hitting humans is still capped at 30 rage from hitting 3 humans
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
Makes eviscerating charge into what surreal wanted, made ravage much less clunky as it used a Move() as well as it's bad ability to go diagonal, avoiding certain FOB cade constructions. 

Ravage ability now no longer move's you in the direction of a turf selected; it now just slices enemies in the direction you use it on, can now be used without a two tile range restriction, slicing in the direction of where you used your ability at. It can also hit more than three enemies but rage generation on victim amount is still capped.

### CODE REVIEW

**Comment by Rohesie** [comment]
Condense your changelog. To something like `balance: Eviscerating charge now plows through humans, slicing and knocking them around. Ravagers suffer slowdown upon hitting humans no more, nor to they bounce upon hitting dense turfs.`

**Comment by Rohesie** [comment]
Took the liberty of editing the changelog.

File: code/modules/mob/living/carbon/xenomorph/Castes/Ravager.dm [hunk 1]
```diff
@@ -338,3 +338,17 @@
 
 		M.adjustFireLoss(rand(20, 50) * fire_mod) //Fwoom!
 		to_chat(M, "[isxeno(M) ? "<span class='xenodanger'>":"<span class='highdanger'>"]Augh! You are roasted by the flames!</Sspan>")
+
+/mob/living/carbon/Xenomorph/Ravager/Bump(atom/A)
+	if(throwing && usedPounce) //Must currently be charging to knock aside and slice marines in it's path
+		if(ishuman(A))
+			var/mob/living/carbon/human/unluckydude = A
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/Castes/Ravager.dm]:
```suggestion
			var/mob/living/carbon/human/H = A
```

File: code/modules/mob/living/carbon/xenomorph/Castes/Ravager.dm [hunk 2]
```diff
@@ -338,3 +338,17 @@
 
 		M.adjustFireLoss(rand(20, 50) * fire_mod) //Fwoom!
 		to_chat(M, "[isxeno(M) ? "<span class='xenodanger'>":"<span class='highdanger'>"]Augh! You are roasted by the flames!</Sspan>")
+
+/mob/living/carbon/Xenomorph/Ravager/Bump(atom/A)
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/Castes/Ravager.dm]: convert this to early returns

**Pull request merged**